gdal/src/main/scala/geotrellis/contrib/vlm/gdal/GDALDataset.scala

```diff
@@ -145,7 +145,7 @@ case class GDALDataset(token: Long) extends AnyVal {
       val minmax = Array.ofDim[Double](2)
       val success = Array.ofDim[Int](1)
       if (GDALWarp.get_band_min_max(token, dataset, numberOfAttempts, 1, true, minmax, success) <= 0)
-        throw new Exception("get_band_min_max")
+        throw new MalformedDataTypeException("Bad min/max values")
```

Comment:
What is the difference between `unable` and `bad` words you use in different exceptions?

Reply:
Really the only difference is that the `Unable` implies that something is `bad`. However, I think it'd be better if we tried to standardized the messages more, so I'll change them to one phrasing.